attributes/default.rb

```diff
@@ -1,18 +1,27 @@
-default['android-sdk']['name']                      = 'android-sdk'
-default['android-sdk']['owner']                     = 'root'
-default['android-sdk']['group']                     = platform_family?('mac_os_x') ? 'wheel' : 'root'
-default['android-sdk']['backup_archive']            = false # The number of backups to be kept in /var/chef/backup. To prevent backups set to false.
-default['android-sdk']['setup_root']                = nil   # ark defaults (/usr/local) is used if this attribute is not defined
-default['android-sdk']['with_symlink']              = true  # use ark's :install action when true; use ark's :put action when false
-default['android-sdk']['set_environment_variables'] = true
+default['android-sdk']['name']                        = 'android-sdk'
+default['android-sdk']['owner']                       = 'root'
+default['android-sdk']['group']                       = platform_family?('mac_os_x') ? 'wheel' : 'root'
+default['android-sdk']['backup_archive']              = false # The number of backups to be kept in /var/chef/backup. To prevent backups set to false.
+default['android-sdk']['setup_root']                  = nil   # ark defaults (/usr/local) is used if this attribute is not defined
+default['android-sdk']['with_symlink']                = true  # use ark's :install action when true; use ark's :put action when false
+default['android-sdk']['set_environment_variables']   = true
 
-default['android-sdk']['version']                   = '24.4'
-default['android-sdk']['checksum']['linux']         = 'f2bb546534d16e2004665257ee530060338c684adad14a49cd4bbde08098d8a4'
-default['android-sdk']['checksum']['mac_os_x']      = 'a1cb0c9b2036d597e18986de1bd43918b6113373ff3bf6fdb81b6cce4ec3efd7'
-default['android-sdk']['checksum']['windows']       = '1f9f22271fe55bf0bf10b60556ed1a5fa2f2ca672fc20d35478325104941cb05'
-default['android-sdk']['download_url']['linux']     = "http://dl.google.com/android/android-sdk_r#{node['android-sdk']['version']}-linux.tgz"
-default['android-sdk']['download_url']['mac_os_x']  = "https://dl.google.com/android/android-sdk_r#{node['android-sdk']['version']}-macosx.zip"
-default['android-sdk']['download_url']['windows']   = "https://dl.google.com/android/android-sdk_r#{node['android-sdk']['version']}-windows.zip"
+default['android-sdk']['version']                     = '25.2.3'
+default['android-sdk']['checksum']['linux']           = '1b35bcb94e9a686dff6460c8bca903aa0281c6696001067f34ec00093145b560'
+default['android-sdk']['checksum']['mac_os_x']        = '593544d4ca7ab162705d0032fb0c0c88e75bd0f42412d09a1e8daa3394681dc6'
+default['android-sdk']['checksum']['windows']         = '23d5686ffe489e5a1af95253b153ce9d6f933e5dbabe14c494631234697a0e08'
+
+if ::Gem::Version.new(node['android-sdk']['version']) < ::Gem::Version.new('25')
```

Comment:
We check the SDK version and set `node['android-sdk']['legacy_sdk']` attribute which will be used in our recipies. Also there are different urls.